frontend/sass/styles.scss

```diff
@@ -1,3 +1,5 @@
+@import '../../node_modules/uswds/dist/css/uswds.css';
```

Comment:
Yeah, I don't know of a better way. If for some reason the path changes in a future release, the webpack build will fail, so it should be "safe" to use this.